uint8 mode
---
bool status